### ISSUE TITLE
Palisades Interstate Parkway Commission shields

### ISSUE THREAD

**Issue by claysmalley** [osm-americana/openstreetmap-americana#517]
Parkways of the Palisades Interstate Parkway Commission have a white-on-brown circular shield, with an illustration of a leaf on top and the full name of the parkway spelled out on bottom.

<img height="150" alt="Palisades_Interstate_Pkwy" src="https://user-images.githubusercontent.com/1732117/181242027-167ea25e-d326-4972-9deb-bcd9be66bef4.svg" />

Currently, the routes in New York are tagged with `network=US:NY:Parkway:PIPC` and the route in New Jersey is tagged `network=US:NJ`. A consistent tagging scheme will have to be devised and implemented before shield support can be added.

**Comment by claysmalley** [comment]
I agree, though right now it seems like shields prevent the rendering of street names. We need to show the name alongside the shield if the shield isn't enough to determine which route it is.

**Comment by ZeLonewolf** [comment]
Wondering if there's a technical solution that lets the street name always appear under the shield.  Alternately, maybe it's possible to spread out the shields more for routes where we know we want the name to be drawn.

**Comment by 1ec5** [comment]
Option 5 in https://github.com/ZeLonewolf/openstreetmap-americana/pull/119#issuecomment-1031143629 might apply here: decrease the minimum zoom level at which roads in this network are labeled, so that the road’s effective “shield” is a text label.

**Comment by claysmalley** [comment]
Good work! This should be ready for an icon. Feel free to submit a PR; otherwise I might get around to doing it myself later this week.
